cms/djangoapps/contentstore/views/item.py

```diff
@@ -1046,6 +1054,15 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         else:
             xblock_info["staff_only_message"] = False
 
+    if is_concise:
```

Comment:
It looks like most of the code before this is unnecessary as it is building up an ``xblock_info`` dict and then replacing it. I think this ``if`` should be earlier before any code starts updating the ``xblock_info``.